duktape/src/main/java/com/squareup/duktape/Duktape.java

```diff
@@ -125,8 +125,11 @@ private static void checkSignatureSupported(Method method) {
 
   /** Returns true if we support {@code: type} in calls from JavaScript. */
   private static boolean isSupportedType(Class<?> type) {
-    // TODO: support some more types.
-    return void.class.equals(type) || String.class.equals(type);
+    return void.class.equals(type)
+        || boolean.class.equals(type)
+        || int.class.equals(type)
+        || double.class.equals(type)
```

Comment:
Actually we probably want to get `long`, `float, `short`, and `char` in here as well, right? At the very least the first two. Also could be a follow-up. Maybe I'll even attempt it!